VirtoCommerce.Storefront/Domain/Cart/CartBuilder.cs

```diff
@@ -82,12 +82,15 @@ public void LoadOrCreateNewTransientCart(string cartName, Store store, User user
                 //Load cart payment plan with have same id
                 if (store.SubscriptionEnabled)
                {
+                   var productsIds = cart.Items.Select(x => x.ProductId).Distinct().ToArray();
+                   var products = await _catalogService.GetProductsAsync(productsIds, ItemResponseGroup.ItemWithPrices);
```

Comment:
Need to load products with these response groups 
ItemResponseGroup.ItemWithPrices | ItemResponseGroup.ItemWithDiscounts | ItemResponseGroup.Inventory

Reply:
![cartbuildertest](https://user-images.githubusercontent.com/29459286/34252301-be2a17bc-e64b-11e7-83a5-c12160e7a35c.gif)